testing/test.js

```diff
@@ -10,6 +9,7 @@ const tracer = require('../packages/dd-trace').init({
   metricsUrl: 'METRICS_URL',
 });
 
+const express = require('express');
```

Comment:
is this a required change here?

Reply:
It seems to be. As far as I can tell, monkey patches don't get applied until after the tracer is initialized. Or at least not all of them are. Before I moved this, I did not see spans being created for the metrics endpoint.